plugins/module_utils/network/ios/facts/snmp_server/snmp_server.py

```diff
@@ -47,6 +47,8 @@ def get_snmpv3_user_data(self, connection):
         Note: The seperate method is needed because the snmpv3 user data is not returned within the snmp-server config
         """
         _get_snmpv3_user = connection.get("show snmp user")
+        if "%" in _get_snmpv3_user:
```

Comment:
That is an appliance error msg, not a command output. we should not handle it here